Gems/AtomLyIntegration/AtomBridge/Code/Source/AtomDebugDisplayViewportInterface.cpp

```diff
@@ -1016,6 +1016,55 @@ namespace AZ::AtomBridge
         }
     }
 
+    void AtomDebugDisplayViewportInterface::DrawWireCylinderNoEnds(const AZ::Vector3& center, const AZ::Vector3& axis, float radius, float height)
+    {
+        if (m_auxGeomPtr)
+        {
+            const float scale = GetCurrentTransform().RetrieveScale().GetMaxElement();
+            const AZ::Vector3 worldCenter = ToWorldSpacePosition(center);
+            const AZ::Vector3 worldAxis = ToWorldSpaceVector(axis);
+            m_auxGeomPtr->DrawCylinderNoEnds(
+                worldCenter, 
+                worldAxis, 
+                scale * radius, 
+                scale * height, 
+                m_rendState.m_color, 
+                AZ::RPI::AuxGeomDraw::DrawStyle::Line,
+                m_rendState.m_depthTest,
+                m_rendState.m_depthWrite,
+                m_rendState.m_faceCullMode,
+                m_rendState.m_viewProjOverrideIndex
+            );
+        }
+    }
+
+    void AtomDebugDisplayViewportInterface::DrawSolidCylinderNoEnds(
+        const AZ::Vector3& center, 
+        const AZ::Vector3& axis, 
+        float radius, 
+        float height, 
+        bool drawShaded)
```

Comment:
Seems weird to have a whole extra function for this. Why not just have an argument for AZ::RPI::AuxGeomDraw::DrawStyle and default it to Line?

Reply:
It's just matching the existing style in that api. I do agree that the api could do with a little spring cleaning in the future.